bios/boot_sector/README.md

```diff
@@ -4,13 +4,13 @@ This executable needs to fit into the 512-byte boot sector, so we need to use al
 
 ## Build Commands
 
-1. `cargo build --release -Zbuild-std=core --target x86-16bit.json -Zbuild-std-features=compiler-builtins-mem`
-2. `objcopy -I elf32-i386 -O binary target/x86-16bit/release/first_stage target/disk_image.bin
+1. `cargo build --profile=stage-1 -Zbuild-std=core --target ../../i386-code16-boot-sector.json -Zbuild-std-features=compiler-builtins-mem`
+2. `objcopy -I elf32-i386 -O binary ../../target/i386-code16-boot-sector/stage-1/bootloader-x86_64-bios-boot-sector ../../target/disk_image.img`
 
 To run in QEMU:
 
-- `qemu-system-x86_64 -drive format=raw,file=target/disk_image.bin`
+- `qemu-system-x86_64 -drive format=raw,file=../../target/disk_image.bin`
```

Comment:
```suggestion
- `qemu-system-x86_64 -drive format=raw,file=../../target/disk_image.img`
```

Reply:
Good catch. If only you could compile check markdown like rust